### PR TITLE
Typography improvements

### LINKED ISSUE #4: Add more line height for the title
## Steps to Reproduce
1. Go to https://webuild-blog.vercel.app/read?url=https://learnlingo.co/tu-vung-hom-nay-hands-on-experience/

Screen Shot 2021-06-25 at 00 41 41

### PR DESCRIPTION
**Pull request by huytd**
Fixes #4 #5 #6 #11 #12

- Use Vietnamese for the interface
- Use Github Markdown Preview style for blog content
- Some UI enhancements

### CODE REVIEW

File: pages/_app.tsx [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import ProgressBar from '@badrap/bar-of-progress';
 import Router from 'next/router';
+import Link from 'next/link';
```
Comment (monodyle) [inline pages/_app.tsx:3]: Unused import

File: pages/read.tsx [hunk 2]
```diff
@@ -33,15 +37,36 @@ const ReadPage = ({ article }) => {
       </Head>
 
       <main className={styles.main}>
-        <Link href="/">
-          <Button icon={LeftArrow}>Back to home page</Button>
-        </Link>
+        <div style={{ display: 'flex', justifyContent: 'space-between' }}>
```
Comment (xuanvan229) [inline pages/read.tsx:40]: Can we move style to css. hihi

File: pages/read.tsx [hunk 3]
```diff
@@ -33,15 +37,36 @@ const ReadPage = ({ article }) => {
...
+            target="_blank"
+          >
+            Đọc trên blog của tác giả
+          </Button>
+        </div>
         <DivPx size={16} />
-        <h1>{article.title}</h1>
-        <div dangerouslySetInnerHTML={{ __html: article.content }}></div>
+        <div
+          style={{
```
Comment (xuanvan229) [inline pages/read.tsx:56]: same

File: pages/index.tsx [hunk 4]
```diff
@@ -75,17 +75,19 @@ const Entry = ({ doc }) => {
       }}
     >
       <Tag color={Tag.colors.gray}>{getHostName(doc.link)}</Tag>
-      <Link href={`/read?url=${doc.link}`}>
-        <h3>{doc.title}</h3>
-      </Link>
-      <p>Posted on {new Date(doc.pubDate).toLocaleDateString()}</p>
+      <h3>
+        <Link href={`/read?url=${encodeURIComponent(doc.link)}`}>
+          <a>{doc.title}</a>
```
Comment (monodyle) [inline pages/index.tsx:80]: What the hell is this? `a` tag as default doesn't have pointer cursor, useless.

**Review by monodyle — CHANGES_REQUESTED** [review]

**Comment by lqt93** [comment]
Except inline styles and unused variable, others LGTM.

**Review by monodyle — COMMENTED** [review]
I prefer to rename `typography.css`. Now it not only typography, out of scope.

**Comment by xuanvan229** [comment]
LGTM!

**Comment by huytd** [comment]
Fix xong rồi mấy ông thần, vào review tiếp nào.

**Review by monodyle — APPROVED** [review]
LOTM

**Review by xuanvan229 — APPROVED** [review]

**Pull request merged**